desktop/core/src/desktop/lib/conf.py

```diff
@@ -661,6 +666,19 @@ def coerce_csv(value):
   raise Exception("Could not coerce %r to csv array." % value)
 
 
+def coerce_list(value) -> list:
+  if isinstance(value, str):
+    try:
+      result = ast.literal_eval(value)
```

Comment:
@sreenaths 
There is still risk of memory exhaustion. What is the size of input expected? Limiting the size?
Take a look this: https://stackoverflow.com/questions/4710247/python-3-are-there-any-known-security-holes-in-ast-literal-evalnode-or-string

Reply:
I had similar thought, but as this is an admin configuration felt its fine. Will see if there are any safer way to implement this.